testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/SocialLoginTest.java

```diff
@@ -129,6 +130,7 @@
         STACKOVERFLOW("stackoverflow", StackOverflowLoginPage.class),
         OPENSHIFT("openshift-v3", OpenShiftLoginPage.class),
         OPENSHIFT4("openshift-v4", OpenShiftLoginPage.class),
+        OPENSHIFT4_BOXED("openshift-v4", "openshift-v4-boxed", OpenShiftLoginPage.class),
```

Comment:
It's not actually "boxed". KubeAdmin as it was before was more descriptive. ;) We need a special config just for the kube:admin.

"Boxed" can by even the normal OpenShift4 test (if the cluster is configured to have more than 1 IdP, which will be the case for the QE test cluster).

Reply:
changed to OPENSHIFT4_KUBE_ADMIN